numpy/testing/utils.py

```diff
@@ -44,6 +44,28 @@ class KnownFailureException(Exception):
 verbose = 0
 
 
+def import_nose():
+    """ Import nose only when needed.
+    """
+    fine_nose = True
```

Comment:
Ah, actually fine is correct, it means "is nose fine" (i.e. new enough version).